src/Components/Patient/ManagePatients.tsx

```diff
@@ -533,7 +533,6 @@ export const PatientManager = () => {
                     </span>
                   </span>
                   <span className="w-full truncate px-1 text-center text-base font-bold">
-                    {patient?.last_consultation?.current_bed?.bed_object.name}
                     <span className="tooltip-text tooltip-bottom">
                       {
                         patient?.last_consultation?.current_bed?.bed_object
```

Comment:
Shouldn't you remove this duplicate tooltip too?

Delete the entire duplicate span node.

Reply:
so should i remove the tooltip ??

Reply:
![image](https://github.com/coronasafe/care_fe/assets/98876115/f199a938-7aef-4715-a599-f1ce44c2c368)

currently, I edited the code for the tooltip to look like this when the mouse hovers.

The code is 
```
<div className="tooltip flex h-full flex-col items-center justify-center">
                  <span className="w-full truncate px-1 text-center text-sm text-gray-900">
                    {
                      patient?.last_consultation?.current_bed?.bed_object
                        ?.location_object?.name
                    }
                  </span>
                  <span className="w-full truncate px-1 text-center text-base font-bold">
                    {patient?.last_consultation?.current_bed?.bed_object?.name}
                  </span>
                  <span className="tooltip-text tooltip-bottom">
                    {
                      patient?.last_consultation?.current_bed?.bed_object
                        ?.location_object?.name
                    }
                    <br />
                    {patient?.last_consultation?.current_bed?.bed_object?.name}
                  </span>
                </div>
```
so should I keep the tooltip or do I entirely remove it?

Reply:
Sure!